### PR TITLE
Flux integration

### PR DESCRIPTION
**Pull request by mbreining**
@justin808 Please review. I added support for Flux (https://github.com/goatslacker/alt). The functionality remains the same.

Note that the latest master has been merged into this branch.

Also note that the alt library does not get installed by `npm install` although it is defined in package.json. I suspect there is a dependency conflict somewhere, although I would expect npm to error out. I didn't get a chance to investigate so for now I'm installing it manually `cd client && npm install alt`.

### CODE REVIEW

File: client/assets/javascripts/components/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  propTypes: {
+    author: React.PropTypes.string.isRequired,
+    text: React.PropTypes.string.isRequired
+  },
+
+  render() {
+    const rawMarkup = marked(this.props.text);
+    return (
+      <div className="comment">
+        <h2 className="commentAuthor foobar">
```
Comment (justin808) [inline client/assets/javascripts/components/Comment.jsx]: foobar?

File: client/server.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ server.app.post('/comments.json', function(req, res) {
   console.log('Just got done with nap!');
   comments.push(req.body.comment);
   res.setHeader('Content-Type', 'application/json');
-  res.send(JSON.stringify(req.body.comment));
+  res.send(JSON.stringify(comments));
```
Comment (justin808) [inline client/server.js]: send back all comments? OK I guess if we knew the list was limited.
I have it coded to send back just the comment that's saved.

File: client/webpack.common.config.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ var path = require('path');
 
 module.exports = {
   context: __dirname, // the project dir
-  entry: ['./assets/javascripts/example'],
+  entry: ["./assets/javascripts/App"],
```
Comment (justin808) [inline client/webpack.common.config.js]: better

**Comment by fredericgrati** [comment]
I have the following error when I tried to go on localhost:4000
```
Uncaught Error: Cannot find module "../actions/FormActions"
```
Everything works fine on localhost:3000.

Thx for your alt integration. Awesome work !

**Comment by mbreining** [comment]
@fredericgrati This PR is no longer valid. I will close it shortly. Check out PR #28 instead. Thanks.

**Pull request closed without merging**